addons/mail/static/src/views/web/activity/activity_renderer.js

```diff
@@ -92,21 +109,31 @@ export class ActivityRenderer extends Component {
             activeBar: isColumnFiltered ? this.activeFilter.progressValue.active : null,
         };
         for (const [value, count] of Object.entries(types)) {
-            progressBar.bars.push({
-                count: count.value,
-                value,
-                string: this.props.fields.activity_state.selection.find((e) => e[0] === value)[1],
-                color: count.color,
-            });
+            if (count.inProgressBar) {
+                progressBar.bars.push({
+                    count: count.value,
+                    value,
+                    string: types[value].label,
+                    color: count.color,
+                });
+            }
         }
 
+        const ongoingActivityCount = types.overdue.value + types.today.value + types.planned.value;
```

Comment:
the file follow lint rules defined in /web/tooling/eslintrc.json before your change, can you check it's still the case ?

Reply:
Thanks, the linter is now working well in my ide.